srml/staking/src/lib.rs

```diff
@@ -1117,7 +1117,8 @@ impl<T: Trait> Module<T> {
 	fn new_session(session_index: SessionIndex)
```

Comment:
the `session_index` here is the session index that the the return value (i.e. the potentially new validator set) will be applied at, and *NOT* the current (ending) session index. really, the name of the parameter and ideally the docs should be changed to reflect this.